std/math.d

```diff
@@ -2181,11 +2181,12 @@ creal expi(real y) @trusted pure nothrow @nogc
 T frexp(T)(T value, out int exp) @trusted pure nothrow @nogc
     if(isFloatingPoint!T)
 {
-    ushort* vu = cast(ushort*)&value;
+    Unqual!T vf = value;
```

Comment:
The `value` can be const or immutable. Should it be `in`: `(in T value, out int exp)`?